src/components/sharing/UserTable.js

```diff
@@ -91,59 +81,8 @@ function UserTable(props) {
         ],
         [baseId, t, onPermissionChange, onRemoveUser, classes.listItem]
     );
-    const { getTableProps, headerGroups, rows, prepareRow } = useTable(
-        {
-            columns,
-            data,
-        },
-        useSortBy
-    );
 
-    return (
-        <Table size="small" stickyHeader {...getTableProps()}>
-            <TableHead>
-                {headerGroups.map((headerGroup) => (
-                    <TableRow {...headerGroup.getHeaderGroupProps()}>
-                        {headerGroup.headers.map((column) => (
-                            <TableCell
-                                padding="none"
-                                {...column.getHeaderProps(
-                                    column.getSortByToggleProps()
-                                )}
-                            >
-                                {column.render("Header")}
-                                <TableSortLabel
-                                    active={column.isSorted}
-                                    direction={
-                                        column.isSortedDesc ? "desc" : "asc"
-                                    }
-                                />
-                            </TableCell>
-                        ))}
-                    </TableRow>
-                ))}
-            </TableHead>
-            <TableBody>
-                {rows.map((row) => {
-                    prepareRow(row);
-                    return (
-                        <TableRow {...row.getRowProps()}>
-                            {row.cells.map((cell) => {
-                                return (
-                                    <TableCell
-                                        padding="none"
-                                        {...cell.getCellProps()}
-                                    >
-                                        {cell.render("Cell")}
-                                    </TableCell>
-                                );
-                            })}
-                        </TableRow>
-                    );
-                })}
-            </TableBody>
-        </Table>
-    );
```

Comment:
All of this ☝️ was pretty much copy-pasted into `BasicTable`, so other components can hopefully reuse it.  The component name can obviously be changed.